ArchiSteamFarm/Program.cs

```diff
@@ -66,12 +68,20 @@ internal static class Program {
 				return null;
 			}
 
+			string result;
+
 			if (GlobalConfig.Headless || !Runtime.IsUserInteractive) {
+				var userInputKey = new KeyValuePair<string, ASF.EUserInputType>(botName, userInputType);
+				if (UserInputs.ContainsKey(userInputKey)) {
```

Comment:
I don't like this, maybe it'd make much more sense to restructurize this into ```Bot``` function that sets given ```Bot``` field based on ```ASF.EUserInputType``` (```HandleUserInput()```?), and change all current calls from ```Bot```->```Program.GetUserInput()``` in a way that we'll ask for value like usual, but then call the same ```HandleUserInput()``` with given ```ASF.EUserInputType``` and value, then in the same way we'd call it in ```!input``` command.

Then the overall flow will be better because:
- one function sets variables, one switch with ```ASF.EUserInputType```
- command calls that function directly with type parsed from user and value parsed from user too
- normal calls call ```Program.GetUserInput()``` like usual, but then instead of assigning this value to a field, they call the same set function that is responsible for that.

What do you think? I can do that if you don't want to or you're confused about what I mean, because overall this PR is good, it could just be done a bit better so we don't introduce a ```Dictionary``` queue, but set variables directly.